Dfe.Academies.External.Web/Pages/School/SchoolConversionKeyDetails.cshtml.cs

```diff
@@ -77,21 +77,80 @@ private void PopulateUiModel(SchoolApplyingToConvert selectedSchool)
+					SchoolConversionComponentStatus.Complete
+					: SchoolConversionComponentStatus.NotStarted
+				};
 
 			// TODO MR:- set from API data
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsHeadteacherNameSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsHeadteacherEmailSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsHeadteacherTelNoSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsChairNameSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsChairEmailSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsChairTelNoSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsMainContactWhomSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsApproversFullNameSectionName, "TBC"));
-			heading2.Sections.Add(new(SchoolConversionComponentSectionViewModel.ContactDetailsApproversEmailSectionName, "TBC"));
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsHeadteacherNameSectionName, 
+					selectedSchool.SchoolConversionContactHeadName ?? QuestionAndAnswerConstants.NoAnswer)
+				);
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsHeadteacherEmailSectionName, 
+					selectedSchool.SchoolConversionContactHeadEmail ?? QuestionAndAnswerConstants.NoAnswer)
+				);
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsHeadteacherTelNoSectionName, 
+					selectedSchool.SchoolConversionContactHeadTel ?? QuestionAndAnswerConstants.NoAnswer)
+				);
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsChairNameSectionName, 
+					selectedSchool.SchoolConversionContactChairName ?? QuestionAndAnswerConstants.NoAnswer)
+				);
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsChairEmailSectionName, 
+					selectedSchool.SchoolConversionContactChairEmail ?? QuestionAndAnswerConstants.NoAnswer)
+				);
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsChairTelNoSectionName, 
+					selectedSchool.SchoolConversionContactChairTel ?? QuestionAndAnswerConstants.NoAnswer)
+				);
+			heading2.Sections.Add(
+				new(
+					SchoolConversionComponentSectionViewModel.ContactDetailsMainContactRoleSectionName, selectedSchool.SchoolConversionContactRole ?? QuestionAndAnswerConstants.NoAnswer )
+				);
+			if (!string.IsNullOrEmpty(selectedSchool.SchoolConversionContactRole) && selectedSchool.SchoolConversionContactRole.Equals(MainConversionContact.Other.ToString()))
+			{
```

Comment:
hi @RobertM905 , where we output the 'Who is the main contact for the conversion?' can this be the enum description rather than the enum raw value of 'HeadTeacher'